src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1130,6 +1190,11 @@ protected async Task<TResult> InvokeCoreAsync<TResult>(long? id, string targetNa
                                     this.TraceSource.TraceEvent(TraceEventType.Warning, (int)TraceEvents.RequestAbandonedByRemote, "Aborting pending request \"{0}\" because the connection was lost.", id);
                                 }
 
+                                if (JsonRpcEventSource.Instance.IsEnabled())
+                                {
+                                    JsonRpcEventSource.Instance.LogWarning($"Aborting pending request \"{id}\" because the connection was lost.");
```

Comment:
It looks like you're duplicating several trace messages, but not all of them.
How did you pick which ones to trace to both places? 
Why is it valuable to use ETW to trace this when we already have a TraceSource?
If it's valuable, couldn't someone just add a trace listener and log ETW from there to avoid the duplication here?

Reply:
Removed them as per yesterday's discussion.